plugins/tours/tests/phpunit/integration/admin/changeTitlePlaceholderText.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Tests for change_title_placeholder_text().
+ *
+ * @package     spiralWebDb\CornerstoneTours\Tests\Integration
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\CornerstoneTours\Tests\Integration;
+
+use spiralWebDb\Cornerstone\Tests\Integration\Test_Case;
+
+/**
+ * Class Tests_ChangeTitlePlaceholderText
+ *
+ * @package spiralWebDb\CornerstoneTours\Tests\Integration
+ * @group   tours
+ * @group   admin
+ */
+class Tests_ChangeTitlePlaceholderText extends Test_Case {
+
+	/**
+	 * Set up each test.
+	 */
+	public function setUp() {
+		parent::setUp();
+	}
+
+	/**
+	 * Test change_title_placeholder_text() should return the given text when post type is 'post'.
+	 */
+	public function test_should_return_given_text_when_post_type_is_not_tours() {
+		// Create and get the post object via the factory method with 'post' post_type.
+		$post = self::factory()->post->create_and_get();
+		$expected = __( 'Add title' );
```

Comment:
You can get rid of the i18n `__()`. Not needed here or in the filter.

Reply:
@hellofromtonya Removed.